src/opnsense/scripts/dhcp/get_leases6.py

```diff
@@ -148,7 +148,7 @@ def parse_lease(lines):
                 elif len(line) > 1 and line[0] == '}' and len(cur_lease) > 0:
                     cur_lease.append(line)
                     parsed_lease = parse_lease(cur_lease)
-                    last_leases[parsed_lease['iaid_duid']] = parsed_lease
+                    last_leases[parsed_lease['lease_type'], parsed_lease['iaid_duid']] = parsed_lease
```

Comment:
```suggestion
                    last_leases["%s,%s" % (parsed_lease['lease_type'], parsed_lease['iaid_duid'])] = parsed_lease
```

I prefer to use single value keys for debugging, the suggested change should have the same result as you're looking for.